nin/backend/nin

```diff
@@ -63,6 +64,29 @@ program
     generate.generate(projectRoot, resource, name);
   });
 
+program
+  .command('update')
+  .description('Update nin to newest version')
+  .action(function() {
+    echo('Checking for new version of nin');
+
+    // normalize nin path to point to /path/to/nin-project/nin
+    var ninExecutable = __dirname;
+    var ninPath = ninExecutable.split('/');
+    var ninPath = ninPath.splice(0, ninPath.length-2).join('/');
+
+    cd(ninPath);
```

Comment:
I guess these functions are magically imported via shelljs, but I was thrown off at first. Perhaps there is a way to import them in a more explicit fashion, so that one would use e.g. `shelljs.cd` or something?